LocalSettings.php

```diff
@@ -701,6 +701,7 @@
 	],
 	'wgCreateWikiCategories' => [
 		'default' => [
+			'Art & Arcitecture' -> 'artarc',
```

Comment:
- should be an =

Reply:
@RhinosF1 Yeah, I noticed that _just_ before you replied, so already fixed it.

Reply:
Seen the fix and approved the change after that change.

Reply:
Wait, don't merge this yet. I noticed another typo.

Reply:
Fixed.